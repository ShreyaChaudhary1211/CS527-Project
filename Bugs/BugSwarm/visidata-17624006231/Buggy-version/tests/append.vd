sheet	col	row	longname	input	keystrokes	comment
			open-file	tests/data1.tsv	o	open input in VisiData
			open-file	tests/data2.tsv	o	open input in VisiData
data2			sheets-stack		S	open Sheets Sheet
sheets		キdata2	select-row		s	select current row
sheets		キdata1	select-row		s	select current row
sheets			join-sheets	append	&	merge the selected sheets with visible columns from all, keeping rows according to jointype
data2&data1	Key		key-col		!	toggle current column as a key column
data2&data1	A		slide-left		H	slide current column left
data2&data1	A		slide-left		H	slide current column left
data2&data1	B		slide-left		H	slide current column left
data2&data1	B		slide-left		H	slide current column left
data2&data1	A		key-col		!	toggle current column as a key column
data2&data1	A		type-string		~	set type of current column to str
data2&data1			sort-keys-asc		g[	sort ascending by all key columns
data2&data1	A		key-col		!	toggle current column as a key column
data2&data1			columns-sheet		C	
data2&data1_columns	width	キorigin_sheet	edit-cell	10	e	
data2&data1_columns		キorigin_sheet	slide-down		J	
data2&data1_columns			quit-sheet		q	
