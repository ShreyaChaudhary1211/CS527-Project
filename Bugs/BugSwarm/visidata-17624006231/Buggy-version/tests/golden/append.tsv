Key	origin_sheet	A	B	C	D
1	data1	a1	b1		
2	data2			a2	b2
2	data1	c1	d1		
2	data1	e1	f1		
3	data2			c2	d2
